src/gengowatcher/state.py

```
@@ -13,6 +13,13 @@
 class AppState:
     STATE_FILE = "state.json"
     MAX_STORED_JOBS = 5000
+    MAX_SEEN_JOB_IDS = 50000
+    SENSITIVE_PERSISTED_JOB_KEYS = {
+        "_raw",
+        "source_text",
+        "accepted_source_text",
+        "segments",
+    }
```

Ruff (0.15.20)

[warning] Mutable default value for class attribute

(RUF012)